_rules/device-motion-disabled-c249d5.md

```diff
@@ -0,0 +1,308 @@
+---
+id: c249d5
+name: Device motion can be disabled
+rule_type: atomic
+description: |
+  This rule checks that it is possible to disable functionality that can be operated by device motion.
+accessibility_requirements:
+  wcag21:2.5.4: # Motion Actuation (A)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+input_aspects:
+  - DOM Tree
+  - CSS Styling
+  - Accessibility tree
+acknowledgments:
+  authors:
+    - João Vicente
+    - Carlos Duarte
+htmlHintIgnore:
+  # https://www.npmjs.com/package/htmlhint
+  # (used with `npm test` to ensure validity of code snippets)
+  - 'attr-lowercase'
+---
+
+## Applicability
+
+The rule applies to an [HTML document][] with an associated [Window object][] that has an [event listener list][] with one or more [event listeners][event listener] for [device orientation events][device orientation] or [device motion events][device motion].
```

Comment:
You're making a rather substantial assumption here that just because there's an event handler, something is actually going to happen in response to the event.

Reply:
I think that is a fair assumption to make (otherwise, why is the event handler there?)
But I agree that we need to add an assumption that what will happen in response to the event is a functionality of the in the web page and not other unrelated result.

Reply:
I disagree, see my comment here: https://github.com/act-rules/act-rules.github.io/pull/1276/files#r434474714

Reply:
Reworked the expectation to take into account the possibility of the event not producing any results.